launch/app/runtime/services.cfg

```diff
@@ -18,7 +18,8 @@ threadpool:safeCall=3
 
 # Start level definition
 startlevel:20=dsl:items,managed:item,dsl:things,managed:thing,managed:itemchannellink,dsl:persist
-#startlevel:30=persistence:restore // do not rely on the presence of a persistence service in the IDE
+startlevel:20=dsl:items,managed:item,dsl:things,managed:thing,managed:itemchannellink,dsl:persist,managed:metadata
+startlevel:30=persistence:services,persistence:restore,automation:scriptEngineFactories
```

Comment:
What about the current comment ?
In IDE, we should be able to start openHAB even without any persistence installed.
Your change should not change that.

Reply:
Hum, don't look at the comment, sorry.

So I don't know what are good modification to solve this.
The matter is that we add new marker as part of @J-N-K modification on September 9.
And we except to openhab go to level 30 to have ThingManagerImpl correctly initialized !

Laurent.

Reply:
Laurent,

This is related to the issue : https://github.com/openhab/openhab-core/issues/3823.
Don't know if you see my comment this morning about tracing why we have thing not get initialized in development environment.

Reply:
Take care, you have two different lines for start level 20.

Reply:
IMO you. eed to remove the second line for start level 20 and should remove the `persistence:restore` marker. The other two markers rely on bundle trackers and will be set when all installed bundles (persistence or scripting) are `ACTIVE` or no bundle of the given type is installed. `persistence:restore` in principle expects a persistence bundle to be installed which might not be the case in a development environment.